src/WebJobs.Script.WebHost/FileMonitoringService.cs

```diff
@@ -93,16 +94,6 @@ public Task StopAsync(CancellationToken cancellationToken)
         /// </summary>
         private void InitializeFileWatchers()
         {
-            _debugModeFileWatcher = new AutoRecoveringFileSystemWatcher(_hostLogPath, ScriptConstants.DebugSentinelFileName,
```

Comment:
Why is simply creating these watchers in ctor an issue? We create watchers in this manner in other classes on construction

Reply:
That's not the main issue, just needed to delay the creation of watchers that can wait until Host is started up. I moved this call to put a delay on them

Reply:
Why though? You have proof this is a problem? What about the other file watchers we're creating on construction - why aren't those a problem?

Reply:
Hamid can provide more context. But we are making platform improvements, and for that the ask is - we need to try to avoid any filesystem calls that are not in wwwroot. The other watchers I believe are for wwwroot. If that's not the case, then we'd have to delay those too.